src/watch/hearRingiOSWatchOS/hearRingiOSWatchOS WatchKit Extension/Sound/Record.swift

```diff
@@ -16,9 +16,13 @@ class Record : NSObject, AVAudioRecorderDelegate{
     var chunking: Chunking? = nil
     let vibration = Vibration()
     
-    init(chunker: Chunking? = nil)
-    {
-        chunking = chunker!
+    DispatchQueue.global.aync {
```

Comment:
I don't understand why you make a thread for the creation of the recording on appear and then one for the initializer. Is this required? Can you explain further for me (maybe leave a comment to explain as well in the code)

Reply:
I thought this was required because of scope issues (i.e., I'm not sure what all "global" applies to, and whether it transfers across pages.) I can test to see whether it is or isn't.

Reply:
Ok, the GCD queue seems to carry across files and declarations.